.pre-commit-config.yaml

```diff
@@ -46,6 +46,9 @@ repos:
       - id: fix-byte-order-marker
       - id: check-case-conflict
       - id: check-toml
+      - id: file-contents-sorter
+        files: dictionary.*\.txt$
+        args: [--ignore-case]
```

Comment:
We didn't have the ignore case option passed to sort before.

Reply:
It was. `SORT_ARGS`, defined in the Makefile and passed to `sort` when called, includes `-f`, which is the supremely non-obvious equivalent of sort's `--ignore-case` argument.